core/lib/workarea/configuration/redis.rb

```diff
@@ -67,8 +71,14 @@ def db
         @config[:db]
       end
 
+      def to_h
```

Comment:
I'm curious, why switch this to be only a URL? How would I get a hash-version of the config now?

Reply:
Ah, I missed your comment above. I think for compatibility it'd be best to keep the hash version around, can you expand the hash to include all the things?

Reply:
Updated